Tests/IntegrationTests.Shared/PerformanceTests.cs

```diff
@@ -133,6 +133,37 @@ public void BindingSetValuePerformanceTest(int count)
             Console.WriteLine("Kilo-iterations per second: {0:0.00}", ((count/1000) / sw.Elapsed.TotalSeconds));
         }
 
+        string benchmarkString;
+
+        [Test, Explicit("Temporary benchmarker")]
```

Comment:
That test should probably be removed. It was mostly to get a feel for the change, but it's not a very reliable performance test in general..